code/ui/manager/src/components/sidebar/Tree.tsx

```diff
@@ -133,6 +133,7 @@ export const LeafNodeStyleWrapper = styled.div(({ theme }) => ({
   '&:hover, &:focus': {
     outline: 'none',
     background: theme.background.hoverable,
+    color: theme.color.lighter,
```

Comment:
Thanks a lot for your changes. I checked the branch locally and the result doesn't seem to work properly. When I hover a story the text is becoming white because you set the hover to use `color: theme.color.lighter`.
What was your intent by doing that?

![CleanShot 2023-09-19 at 09 39 32](https://github.com/storybookjs/storybook/assets/1540635/abbe0c2f-295a-4504-a58c-21e32e90334b)

Reply:
Oh, Pardon me. I didn't test for the light mode to see this.
In dark mode, the color of the text on hover and focus was lighter. Which made it pop and more noticable.
Which gave a good contrast ratio score as well.

However, it wasn't so complete